.gitignore

```diff
@@ -29,5 +29,4 @@ coverage.xml
 
 # Test files
 requirements.in
-requirements.txt
```

Comment:
Don't remove this. It's necessary to exclude the test files. Instead, add a new gitignore in the docs dir reincluding this entry.

Reply:
okay, i try